test/build.gradle.kts

```diff
@@ -38,25 +42,34 @@ subprojects {
             useJUnitPlatform()
         }
     }
-    this.tasks.withType<Test> {
+
+    tasks.withType<Test> {
+        dependsOn(":test:test-server:startServer")
+        environment("host" to host)
+        environment("port" to port)
         dependsOn(project(":").tasks.getByPath("linkAutoTestAgentDebugShared${presetName.capitalize()}"))
         dependsOn(project(":").tasks.getByPath("install${presetName.capitalize()}Dist"))
+        finalizedBy(":test:test-server:killProcess")
     }
 
+    project.extra["adminHost"] = host
+    project.extra["adminPort"] = port
```

Comment:
remove

Reply:
these properties are needed in the admin-stub-server module

Reply:
can we use only project.extra or environments? Do we really need both?

Reply:
Unfortunately, I won't be able to find a way to use one thing, we need environment variables so that the tests know the data about the server, and the project variables are needed to start the server.